cpp/ql/src/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Semmle C/C++ Default Queries
 Bundle-SymbolicName: com.semmle.plugin.semmlecode.cpp.queries;singleton:=true
-Bundle-Version: 1.18.1.qualifier
+Bundle-Version: 1.18.1
 Bundle-Vendor: Semmle Ltd.
 Bundle-ActivationPolicy: lazy
-Require-Bundle: com.semmle.plugin.qdt.ui;bundle-version="[1.18.1.qualifier,1.18.1.qualifier]"
+Require-Bundle: com.semmle.plugin.qdt.ui;bundle-version="[1.18.1,1.18.1]"
```

Comment:
This change looks wrong

Reply:
How come? See e.g. https://github.com/Semmle/ql/commit/f5bd737adaff7cafe10d4dbe5f706a339ee70c5b.

Reply:
Ah indeed, this is `ql:master` not `ql:next`.